### PR TITLE
commas and apostrophes deleted from the descriptions

### PR DESCRIPTION
**Pull request by AnastasiaLobanova**
**Is this a new addition or an update to an existing factor?**

- [ ] New
- [x] Update

**What category or categories does it affect?**

All categories (more or less)

**What?**

This is the final retouching of the DB bf the release. Small things like commas, double quotes, and apostrophes are deleted

**Why?**

Because the descriptions contained commas, that have created problems in the .csv

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -975,16 +975,16 @@ Transport,Air Transport,freight_flight-route_type_outside_uk-distance_unknown-we
...
+Transport,Rail Travel,passenger_train-route_type_international_rail-fuel_source_unknown,International rail passenger train,kg-CO2e/pkm,0.02813,,BEIS,2021,GB,2021/07/23,Emission intensity for international rail passenger train including fuel consumption. The international rail factor is based on a passenger-km weighted average of the conversion factors for the following Eurostar routes: London-Brussels London-Paris London-Marne Le Vallee (Disney) London-Avignon London-Amsterdam and the ski train from London to Bourg-Saint-Maurice. The conversion factors were provided by Eurostar for the 2021 update together with information based on the electricity figures used in their calculation. Published by BEIS in June 2021. ,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,fuel_consumption
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:982]: "London-Brussels London-Paris London-Marne Le Vallee (Disney) London-Avignon London-Amsterdam and the ski train from London to Bourg-Saint-Maurice" -> "London-Brussels/London-Paris/London-Marne Le Vallee (Disney)/London-Avignon/London-Amsterdam/London to Bourg-Saint-Maurice (ski train)"

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -1001,12 +1001,12 @@ Energy,Fuel,fuel_type_marine_fuel_oil,Marine fuel oil,kg-CO2e/l,3.10669,,BEIS,20
...
+Transport,Air Travel,passenger_flight-route_type_unknown-aircraft_type_unknown-distance_medium_haul-class_unknown-contrails_included,Medium-haul flight,kg-CO2e/pkm,0.187,10,ADEME,2018,FR,2021/08/20,Emission intensity of medium-haul international flights to/from/within France including contrails. Includes fuel combustion and upstream emissions. Published by the Agence de lEnvironnement et de la Maîtrise de lÉnergie. Provided by Ecocalculateur DGAC.,https://www.bilans-ges.ademe.fr/en/accueil,fuel_combustion_and_upstream
+Transport,Air Travel,passenger_flight-route_type_unknown-aircraft_type_unknown-distance_between_300_and_2300mi-class_unknown-contrails_unknown,Medium-haul flight,kg-CO2e/pmile,0.131,,EPA,2021,US,2021/08/23,Emission intensity of medium haul flight. Medium Haul (>= 300 miles < 2300 miles). Published by the EPA. Updated April 2021. Accessed on the 23d of July 2021.,https://www.epa.gov/climateleadership/ghg-emission-factors-hub,unknown
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1005]: "medium haul flight. Medium Haul (>= 300 miles < 2300 miles)" -> "medium haul flight (between 300 and 2300 miles)"

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -1042,62 +1042,62 @@ Transport,Vehicle,commercial_vehicle-vehicle_type_hgv-fuel_source_diesel-engine_
...
+Transport,Road Transport,freight_vehicle-vehicle_type_hgv_rigid_refrig-fuel_source_diesel-vehicle_weight_unknown-percentage_load_50_laden,Road freight diesel refrigerated rigid HGV 50% laden,kg-CO2e/tkm,0.26286,,BEIS,2021,GB,2021/07/23,Emission intensity of goods transported by diesel refrigerated rigid HGV with 50% laden. No upstream or downstream LCA activity is included only fuel combustion. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021. All Diesel. ,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,fuel_combustion
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1082]: "No upstream or downstream LCA activity is included only fuel combustion." -> "Emissions from fuel combustion only." throughout DB

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -1159,48 +1159,48 @@ Transport,Vehicle,rental_vehicle-vehicle_type_car-fuel_source_petrol-engine_size
...
+Transport,Rail Travel,passenger_train-route_type_unknown-fuel_source_electricity,Electric passenger train,kg-CO2e/pkm,0.0388,20,ADEME,2020,IE,2021/08/20,Emission intensity for electric train travel (per passenger). This emission factor accounts for energy (electricity - at plant) including fuel combustion and upstream. The data for European countries were taken from a study of the International Union of Rail-ways (UIC) on external effects commonly referred to as the INFRAS-IWW study which was updated in October 2004 with reference year 2000. This study makes it possible to establish emission factors for rail passenger transport in different European countries for traction energy alone. All the available data integrate the upstream and combustion of the traction energy used. It does not include the manufacture of rolling stock or emissions related to the infrastructure (railway tracks stations etc.). Published by the Agence de lEnvironnement et de la Maîtrise de lÉnergie in May 2020. ,https://www.bilans-ges.ademe.fr/en/accueil,combustion_and_upstream
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1169]: "This emission factor accounts for energy (electricity - at plant) including fuel combustion and upstream. The data for European countries were taken from a study of the International Union of Rail-ways (UIC) on external effects commonly referred to as the INFRAS-IWW study which was updated in October 2004 with reference year 2000. This study makes it possible to establish emission factors for rail passenger transport in different European countries for traction energy alone. All the available data integrate the upstream and combustion of the traction energy used. It does not include the manufacture of rolling stock or emissions related to the infrastructure (railway tracks stations etc.)." -> "Including emissions from electricity generation and transmission/distribution" throughout DB

File: OpenEmissionFactorsDB.csv [hunk 5]
```diff
@@ -1211,53 +1211,53 @@ Transport,Sea Transport,sea_freight-vessel_type_overseas_goods_ship-route_type_u
...
-Transport,Rail Travel,passenger_train-route_type_local-fuel_source_diesel,Diesel near passenger train ,kg-CO2e/pkm,0.0713,,UBA,2020,DE,2021/09/15,Emission intensity of diesel local passenger train including upstream. Data from the Öko-Institut's renewability technology database.,https://www.probas.umweltbundesamt.de/php/prozessdetails.php?id={08984DE8-B8F1-46C1-824B-910FD9E8A023},combustion_and_upstream
+Transport,Rail Transport,freight_train-route_type_domestic-fuel_type_electric,Rail freight electric train,kg-CO2e/tkm,0.019,,UBA,2020,DE,2021/09/15,Emission intensity of goods transported by electric rail freight transport. Direct electricity consumption and upstream LCA activity is included. Data from the Öko-Instituts renewbility technology database. Value applicable to 2020.Provided by the Öko-Institut. Accessed on the 15th of September 2021.,https://www.probas.umweltbundesamt.de/php/prozessdetails.php?id={CEB3265F-2B8B-46EA-9166-4BD4F07EAD1F},upstream_and_fuel_combustion
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1214]: Replace "Data from the Öko-Instituts renewbility technology database." -> "Data from the Öko-Institut" thrroughout DB

Replace ".Provided by the Öko-Institut." -> ". Provided by the UBA" throughout DB

**Review by patstirling — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv [hunk 6]
```diff
@@ -609,299 +609,299 @@ Transport,Vehicle,passenger_vehicle-vehicle_type_lower_medium_car-fuel_source_pl
...
-Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kwh,0.40523,,EPA,2018,US-OK,2021/07/23,Emission intensity of grid mix as reported. Available in the EPA's eGrid 2020. It is Generation Factor.,https://www.epa.gov/egrid,generation
-Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kwh,1.02,,DISER,2018,AU-VIC,2021/07/30,Emission Intensity of Victoria's Grid Mix. Published by the Australian Government 2019. Includes Generation Factor.,https://www.industry.gov.au/data-and-publications/national-greenhouse-accounts-factors-2019,generation
+Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kwh,0.0977,,MfE,2019,NZ,2021/09/10,Emission intensity of grid mix as reported. Published by New Zealands Ministry for the Environment. Based on the 2018 national inventory which is based on 2016 data. Includes Production Fuel Mix Factor. It is Generation Factor.,https://www.mfe.govt.nz/node/18670/,production_fuel_mix
```
[the remark this answers is not in the document]
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:772]: @Pat here you mean remove all instances of "It is Generation Factor" from MfE entries?

[the remark this answers is not in the document]
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:772]: Or replace it with "Includes generation factor"?

[the remark this answers is not in the document]
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:772]: Sorry comment was lost in the mist - well spotted, it should be replace with "Includes generation factor" for now

File: OpenEmissionFactorsDB.csv [hunk 7]
```diff
@@ -1159,105 +1159,105 @@ Transport,Vehicle,rental_vehicle-vehicle_type_car-fuel_source_petrol-engine_size
...
+Transport,Rail Travel,passenger_train-route_type_unknown-fuel_source_electricity,Electric passenger train,kg-CO2e/pkm,0.187,20,ADEME,2020,FR,2021/08/20,Including emissions from electricity generation and transmission/distribution. Published by ADEME in May 2020. ,https://www.bilans-ges.ademe.fr/en/accueil,combustion_and_upstream
+Transport,Rail Travel,passenger_train-route_type_transit_rail-fuel_source_unknown,Transit rail passenger train,kg-CO2e/pmile,0.106,,EPA,2021,US,2021/08/23,Emission intensity for average passenger train. This emission factor accounts for transit rail: rail typically within an urban center such as subways elevated railways metropolitan railways (metro) streetcars trolley cars and tramways. Published by the EPA. The method of calculation used is the distance-based method which involves determining the mass distance and mode of each shipment then applying the appropriate mass-distance emission factor for the vehicle used. The activity is unknown. This emission factor was updated in April 2021. ,https://www.epa.gov/climateleadership/ghg-emission-factors-hub,unknown
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1185]: "rail typically within an urban center such as subways elevated railways metropolitan railways (metro) streetcars trolley cars and tramways. Published by the EPA. The method of calculation used is the distance-based method which involves determining the mass distance and mode of each shipment then applying the appropriate mass-distance emission factor for the vehicle used. The activity is unknown." -> "rail typically within an urban center such as subways/elevated railways/metropolitan railways (metro)/streetcars/trolley cars/tramways. Published by the EPA. The method of calculation used is the distance-based method which involves determining the mass/distance/mode of each shipment then applying the appropriate mass-distance emission factor for the vehicle used. LCA activity is unknown.

File: OpenEmissionFactorsDB.csv [hunk 8]
```diff
@@ -1159,105 +1159,105 @@ Transport,Vehicle,rental_vehicle-vehicle_type_car-fuel_source_petrol-engine_size
...
+Transport,Road Transport,freight_vehicle-vehicle_type_transport_light_duty-fuel_source_cng-vehicle_weight_unknown-percentage_load_unknown,Road freight CNG light duty truck average value,kg-CO2e/tkm,0.259,,UBA,2020,DE,2021/09/15,Emission intensity of goods transported by CNG light-duty truck. Upstream and fuel combustion LCA activity is included. Data from the Öko-Institut. Provided by the UBA.,https://www.probas.umweltbundesamt.de/php/prozessdetails.php?id={B258A20B-7767-45E8-A673-829AC523C776},upstream_and_fuel_combustion
+Transport,Air Transport,freight_flight-route_type_domestic-distance_unknown-weight_unknown-contrails_unknown,Domestic air freight,kg-CO2e/tkm,1.19,,UBA,2020,DE,2021/09/15,Emission intensity of inland air cargo in Germany. Fuel combustion and upstream emissions included; detail on contrails not provided. Data from the Öko-Instituts renewability technology database.,https://www.probas.umweltbundesamt.de/php/prozessdetails.php?id={1657B6AD-3B4E-42AE-BFA4-14E53F85F067},fuel_combustion_and_upstream
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv]: "Data from the Öko-Instituts renewability technology database." -> "Provided by the UBA" throughout DB

**Review by patstirling — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 8 above)
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv]: I cannot find this entry in the latest commit..

Comment (greenercode) [reply OpenEmissionFactorsDB.csv]: Btw and renewability should be spelled as '[renewbility](http://www.renewbility.de/)'.

Comment (patstirling) [reply OpenEmissionFactorsDB.csv]: We should (eventually) remove all of these as it is VERY confusing to the reader and doesn't add much - simply "Provided by the UBA" is enough. Adding to data health

File: OpenEmissionFactorsDB.csv [hunk 9]
```diff
@@ -609,299 +609,299 @@ Transport,Vehicle,passenger_vehicle-vehicle_type_lower_medium_car-fuel_source_pl
...
+Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kwh,0.325,,AIB,2019,RU,2021/07/30,Emission intensity of grid mix as reported. Published by the Association of Issuing Bodies (AIB) 2019. Includes production fuel mix factor.,https://www.aib-net.org/,production_fuel_mix
+Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kwh,0.0013,,UNFCCC,2018,CA-QC,2021/07/30,Emission intensity of grid mix as reported. Published in the UN Framework Convention on Climate Change 2020. Includes generation factor.,https://data.ec.gc.ca/data/substances/monitor/canada-s-official-greenhouse-gas-inventory/Emission_Factors.pdf,generation
+Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kwh,0.0977,,MfE,2019,NZ,2021/09/10,Emission intensity of grid mix as reported. Published by Includes production fuel mix factor. Based on the 2018 national inventory which is based on 2016 data. Includes production fuel mix factor. Includes generation factor.,https://www.mfe.govt.nz/node/18670/,production_fuel_mix
```
[the remark this answers is not in the document]
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:772]: It is just this in line! But this is a very nice catch - I got really lucky here, bc the previous comments was:

"New Zealands Ministry for the Environment" -> "the New Zealand MfE" in all cases.

And I have wrongly replaced it with the "Includes production fuel mix factor". Luckily, this is the only place with the "New Zealands Ministry for the Environment", other entries related to MfE have "Published by the Ministry for the Environment of New Zealand" in the description. I will replace it this way also in this line and if we want to change it in the future, I suggest adding it to the Health Card.

File: OpenEmissionFactorsDB.csv [hunk 10]
```diff
@@ -609,299 +609,299 @@ Transport,Vehicle,passenger_vehicle-vehicle_type_lower_medium_car-fuel_source_pl
...
+Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kwh,0.36,10,ADEME,2014,DK,2021/08/13,Emission intensity of grid mix as reported. The values take electrical and thermal kWh supply into account. Cross-border power exchanges are not included. The publication covers only direct emissions from power plants. Provided by the IEA in 2011. Published by ADEME.,https://www.bilans-ges.ademe.fr/en/accueil,fuel_combustion
+Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kwh,0.21233,,BEIS,2021,GB,2021/07/23,Emission intensity of electricity generated as reported. Available in BEISs Greenhouse Gas Reporting: Conversion Factors 2021. These factors include only direct emissions at UK power stations and from autogenerators plus those from the proportion of imported electricity. They do not include emissions resulting from production and delivery of fuel to these power stations.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,electricity_generated
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:660]: "BEISs" -> "BEIS"

**Review by patstirling — CHANGES_REQUESTED** [review]

**Review by patstirling — APPROVED** [review]

**Pull request merged**